compiler/rustc_target/src/spec/armv6k_nintendo_3ds.rs

```diff
@@ -38,7 +38,7 @@ pub fn target() -> Target {
             exe_suffix: ".elf".into(),
             no_default_libraries: false,
             // There are some issues in debug builds with this enabled in certain programs.
-            has_thread_local: false,
+            has_thread_local: true,
```

Comment:
Oops, extra change snuck into the compiler relating to the 3ds target, I would remove this if it's accepted and we move forward with implementation